include/types.hpp

```diff
@@ -135,5 +135,48 @@ using DbusInvalidArgument =
 using InvalidArgument = phosphor::logging::xyz::openbmc_project::Common::InvalidArgument;
 
 namespace DeviceError = sdbusplus::xyz::openbmc_project::Common::Device::Error;
+
+/* PEL Severity enum as defined in xyz.openbmc_project.Logging.Entry.Level */
+enum SeverityType
+{
+    Notice,
+    Informational,
+    Debug,
+    Warning,
+    Critical,
+    Emergency,
+    Alert,
+    Error
+};
+
+/* PEL callout priority from 'phosphor-logging' pel_types.hpp. If any change in 'phosphor-logging', it needs update here as well. */
+enum CalloutPriority
+{
```

Comment:
why is the enum given character values here ?

Reply:
as enum can have literal value, it will hold ASCII value of these char.
Later it could be converted as string.

I removed this & keeping value starting from 0, and having map to represent these values in event_logger.hpp itself